c/compare.sh

```diff
@@ -60,7 +60,7 @@ for f in $SETS ; do
   if [ $setf = ConcurrencySafety-Main ] ; then
     echo "Skipping category $setf (platform-dependent types)"
     continue
-  elif [[ $SKIP_LARGE -eq 1 && $setf == Systems_DeviceDriversLinux64_ReachSafety* ]] ; then
+  elif [[ $SKIP_LARGE -eq 1 && ( $setf == Systems_DeviceDriversLinux64_ReachSafety* || $setf == "Systems_OpenBSD_MemSafety" ) ]]; then
```

Comment:
Why doesn't this work?

Reply:
I added a custom string on the reasons.
As you can tell from the commit history in this PR, I tried but couldn't get it to work
with the nested includes.